Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -104,7 +103,9 @@ class TabManagerImplementation: LegacyTabManager {
         Task {
             // This value should never be nil but we need to still treat it as if it can be nil until the old code is removed
             let activeTabID = UUID(uuidString: self.selectedTab?.tabUUID ?? "") ?? UUID()
-            let windowData = WindowData(activeTabId: activeTabID,
+            // Hard coding the window ID until we later add multi-window support
+            let windowData = WindowData(id: UUID(uuidString: "44BA0B7D-097A-484D-8358-91A6E374451D")!,
```

Comment:
Where does this UUID come from?

Reply:
It's a random UUID. I could also generate one on the fly and store it in user defaults but that seemed like extra unnecessary steps. The main thing is that it needs to be the same everywhere until we add multi window support.